python/fledge/services/core/api/control_service/pipeline.py

```diff
@@ -490,9 +490,25 @@ async def _check_parameters(payload, request):
             if (source_name in south_schedules and des_name in south_schedules) or (
                     source_name in north_schedules and des_name in north_schedules):
                 raise ValueError(error_msg)
+            if source_name in south_schedules:
+                raise ValueError("South services can not be the source for control pipelines")
+            if des_name in north_schedules:
+                raise ValueError("North services can not be the destination for control pipelines")
```

Comment:
From the outset of the initial implementation, this information was not known to us.
As per doc - 

```
The combination of source and destination must be unique for a given pipeline. No two pipelines can have the same source and destination defined, however multiple pipelines can exist with the same source if each has a different destination and likewise multiple pipelines can exist with the same destination if they each have a different source. There can only be one pipeline with a wildcard source and a wildcard destination
```

We should update the DOC as well

Reply:
Doc has been updated

Reply:
Thanks